plugins/module_utils/network/sonic/config/system/system.py

```diff
@@ -550,6 +625,25 @@ def remove_default_entries(self, data):
             adjust_txrx_clock_freq = data.get('adjust_txrx_clock_freq', None)
             if adjust_txrx_clock_freq:
                 new_data["adjust_txrx_clock_freq"] = adjust_txrx_clock_freq
+            login_exec_timeout = data.get('login_exec_timeout', None)
+            if login_exec_timeout is not None and login_exec_timeout != 600:
+                new_data["login_exec_timeout"] = login_exec_timeout
+            new_banner = {}
+            banner = data.get('banner', None)
+            if banner:
+                login_banner_disable = banner.get("login_banner_disable", None)
+                if login_banner_disable is not False:
```

Comment:
Both "False" (default) and "None" should be ignored.
```suggestion
                if login_banner_disable:
```